chgnet/graph/converter.py

```diff
@@ -150,7 +150,7 @@ def forward(
             # Report structures that failed creating bond graph
             # This happen occasionally with pymatgen version issue
             structure.to(filename="bond_graph_error.cif")
-            raise SystemExit(
```

Comment:
Not sure if this is intended, but I don't think `SystemExit` should be raised here?